tests/unit/core/test_utils.js

```diff
@@ -209,6 +209,48 @@ describe('getClientApp', () => {
   });
 });
 
+describe('isAddonAuthor', () => {
+  const addon = {
+    ...fakeAddon,
+    authors: [
+      {
+        id: 5838591,
+        name: 'tofumatt',
+        picture_url: 'http://cdn.a.m.o/myphoto.jpg',
+        url: 'http://a.m.o/en-GB/firefox/user/tofumatt/',
+        username: 'tofumatt',
+      },
+    ],
+  };
+
+  it('returns true when userId is in add-on author object', () => {
+    expect(isAddonAuthor({ addon, userId: 5838591 })).toEqual(true);
+  });
+
+  it('returns true when userId is not in add-on author object', () => {
+    expect(isAddonAuthor({ addon, userId: 5838592 })).toEqual(false);
+  });
+
+  it('returns false when addon is not set', () => {
+    expect(isAddonAuthor({ addon: null, userId: 5838591 })).toEqual(false);
+  });
+
+  it('returns false when addon.authors is not set', () => {
+    expect(isAddonAuthor({ addon: {}, userId: 5838591 })).toEqual(false);
+  });
+
+  it('returns false when userId is not set', () => {
+    expect(isAddonAuthor({ addon, userId: null })).toEqual(false);
+  });
+
+  it('returns false if add-on has no authors', () => {
+    const partialAddon = { ...addon, authors: [] };
```

Comment:
I don't see `authors: null` covered which is in the if branch. However, I don't know if that's very realistic.

Reply:
It will work, so I added a test case. I'm not sure either but good to test it to make sure.